OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -232,171 +230,45 @@ public String getMessage() {
-       */
-      public Builder autoPromptLocation(boolean enable) {
-         mPromptLocation = enable;
-         return this;
-      }
-
-      /**
-       * Prompts the user to update/enable Google Play Services if it's disabled on the device.
-       *
-       * @param disable if {@code false}, prompt users. if {@code true}, never show the out of date prompt.
-       *                Default is {@code false}
-       * @return
-       */
-      public Builder disableGmsMissingPrompt(boolean disable) {
-         mDisableGmsMissingPrompt = disable;
-         return this;
-      }
-
-      public Builder inFocusDisplaying(OSInFocusDisplayOption displayOption) {
-         mDisplayOptionCarryOver = false;
-         mDisplayOption = displayOption;
-         return this;
-      }
-
-      /**
-       * If notifications are disabled for your app, unsubscribe the user from OneSignal.
-       * This will happen when your users go to <i>Settings</i> > <i>Apps</i> and turn off notifications or
-       * they long press your notifications and select "block notifications". This is {@code false} by default.
-       * @param set if {@code false} - don't unsubscribe users<br/>
-       *            if {@code true} - unsubscribe users when notifications are disabled<br/>
-       *            the default is {@code false}
-       * @return the builder you called this method on
-       */
-      public Builder unsubscribeWhenNotificationsAreDisabled(boolean set) {
-         mUnsubscribeWhenNotificationsAreDisabled = set;
-         return this;
-      }
-
-      /**
-       * Enable to prevent other broadcast receivers from receiving OneSignal FCM/GCM payloads.
-       * Prevent thrown exceptions or double notifications from other libraries/SDKs that implement
-       * notifications. Other non-OneSignal payloads will still be passed through so your app can
-       * handle FCM/GCM payloads from other back-ends.
-       * <br/><br/>
-       * <b>Note:</b> You can't use multiple
-       * Google Project numbers/Sender IDs. They must be the same if you are using multiple providers,
-       * otherwise there will be unexpected subscribes.
-       * @param set
-       * @return
-       */
-      public Builder filterOtherGCMReceivers(boolean set) {
-         mFilterOtherGCMReceivers = set;
-         return this;
-      }
-
-      public void init() {
-         OneSignal.init(this);
-      }
-   }
-
-   @Nullable static String appId;
-   private static String mGoogleProjectNumber;
+   static String appId;
+   static String googleProjectNumber;
    static Context appContext;
 
    private static LOG_LEVEL visualLogLevel = LOG_LEVEL.NONE;
    private static LOG_LEVEL logCatLevel = LOG_LEVEL.WARN;
 
-   private static String userId = null, emailId = null;
+   private static String userId = null;
+   private static String emailId = null;
    private static int subscribableStatus;
 
+   // TODO: These should be cleaned up and managed else where maybe?
```

Comment:
👍 Yes, I am ok with doing so in another PR to keep this one smaller.

Reply:
Sounds good to me